src/ert/scheduler/lsf_driver.py

```diff
@@ -134,15 +134,41 @@ async def submit(
         self._iens2jobid[iens] = job_id
 
     async def kill(self, iens: int) -> None:
```

Comment:
I see that now it would be handy to have a return value (bool) from submit and kill cmd.